.github/workflows/cd.yml

```diff
@@ -59,5 +59,46 @@ jobs:
         with:
           identifier: o2sh.onefetch
           installers-regex: '\.exe$'
-          version: ${{ github.event.release.tag_name }}
           token: ${{ secrets.WINGET_TOKEN }}
+
+  cargo-publish:
+    name: Publish to crates.io
+    runs-on: ubuntu-latest
+    env:
+      CARGO_TERM_COLOR: always
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          components: clippy
+
```

Comment:
All of `actions-rs` is deprecated right now. TBH I'm wondering if we actually need a step to install Rust here. GitHub Actions have Rust and `cargo` installed by default, and typically a pretty recent version AFAIK. I never bothered to explicitly install the Rust toolchain in some personal projects, and I've never had a problem.

Reply:
Interesting. Looking more closely, the existing `deploy` job has the following “Install Rust” step:

https://github.com/o2sh/onefetch/blob/ac7e2b8e3313c70f7e7960aefa1296e302ac19e6/.github/workflows/cd.yml#L17-L21

I suppose it makes sense to either match that in `cargo-publish`, or remove *both* “Install Rust” steps. What do you think?

Reply:
Let's add dtolnay's action for consistency.